mlflow/cli.py

```diff
@@ -185,8 +185,7 @@ def run(
     local projects run from the project's root directory.
     """
     if experiment_id is not None and experiment_name is not None:
-        eprint("Specify only one of 'experiment-name' or 'experiment-id' options.")
-        sys.exit(1)
+        raise click.UsageError("Specify only one of 'experiment-name' or 'experiment-id' options.")
```

Comment:
The behavior of a command exiting with `sys.exit(1)` was changed:

https://github.com/pallets/click/issues/2926

Reply:
We have more `sys.exit(1)` in this file, do we need to switch them with `click.UsageError` as well?

Reply:
Yes

Reply:
Replaced a few more `sys.exit(1)`.